src/App.tsx

```diff
@@ -1,11 +1,48 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { MovieData } from './types/MovieData';
+
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [formInput, setFormInput] = useState('');
+  const [newMovies, setNewMovies] = useState<MovieData | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  const changeInput = (value: string) => {
+    setFormInput(value);
+  };
```

Comment:
please check if we can move this or any other methods to the component too. No need to lift a state where it isn't necessary